src/release/breaking-changes/index.md

```diff
@@ -30,12 +30,14 @@ release, and listed in alphabetical order:
 
 ### Not yet released to stable
 
+* [Dart 3 changes in Flutter v3.10 and later][]
 * [Deprecated API removed after v3.7][]
 * [Removed `ignoringSemantics`][]
 * [The window singleton is deprecated][]
 * [Migration guide for `RouteInformation.location`][]
 * [At least one clipboard data variant must be provided][]
 
+[Dart 3 changes in Flutter v3.10 and later]: {{site.url}}/release/breaking-changes/dart3
```

Comment:
Are you going to introduce a page on the Flutter site too or do you mean to link this to the Dart page in the PR mentioned?

```suggestion
[Dart 3 changes in Flutter v3.10 and later]: {{site.dart-site}}/resources/dart-3-migration
```

Reply:
I meant to link to Dart; thanks for catching!